src/environmentd/src/http/sql.rs

```diff
@@ -14,16 +14,37 @@ use serde::Deserialize;
 
 use crate::http::AuthedClient;
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Debug)]
+pub struct SqlRequestParam {
+    value: String,
+}
+
+#[derive(Deserialize, Debug)]
 pub struct SqlRequest {
-    sql: String,
+    query: String,
+    params: Option<Vec<SqlRequestParam>>,
 }
 
 pub async fn handle_sql(
     AuthedClient(mut client): AuthedClient,
-    Json(SqlRequest { sql }): Json<SqlRequest>,
+    Json(request): Json<Vec<SqlRequest>>,
```

Comment:
I'd love to make it possible to specify an array of `SqlRequest` _or_ a single `SqlRequest`. That should make this change wholly backwards compatible with the current HTTP API. I think you can pull this off with a serde untagged union? https://serde.rs/enum-representations.html#untagged